fecgov/fec-cms
# new presidential links on browse data/candidates page

## Summary:
- Add new links to presidential browse data pages
- Edit accordions and language
- Add feature flag to presidential map link


- Resolves #3522


## Impacted areas of the application
	modified:   data/templates/partials/browse-data/candidates.jinja
	new file:   fec/static/img/map-us-states-primary-contrast.svg

## Screenshots
<img width="794" alt="Screen Shot 2020-03-16 at 8 22 43 PM" src="https://user-images.githubusercontent.com/5572856/76810304-f383b480-67c3-11ea-9e33-090bd844fa39.png">


## How to test
- checkout `feature/3522-make-links-to-presidential-map`
- `export  FEC_FEATURE_PRESIDENTIAL_MAP=1`
- `npm run build-sass`
- view in browser to confirm links on http://127.0.0.1:8000/data/browse-data/?tab=candidates
- `unset  FEC_FEATURE_PRESIDENTIAL_MAP` to see first link removed
____
